### ISSUE TITLE
Illegal Argument Exception: Destination offset is greater than stream length

### ISSUE THREAD

**Issue by palm72** [veraPDF/veraPDF-library#1171]
VeraPDF 1.18.6 Greenfield do a full stop/crash when the PDF-file contains an startxref value bigger than the maximum offset in the file: 

```
<buildInformation>
    <releaseDetails id="core" version="1.18.11" buildDate="2021-04-19T10:21:00+02:00"></releaseDetails>
    <releaseDetails id="validation-model" version="1.18.8" buildDate="2021-04-19T10:35:00+02:00"></releaseDetails>
    <releaseDetails id="gui" version="1.18.6" buildDate="2021-04-27T08:53:00+02:00"></releaseDetails>
  </buildInformation>
  <jobsException in thread "main" java.lang.IllegalArgumentException: Destination offset is greater than stream length
        at org.verapdf.io.InternalInputStream.seek(InternalInputStream.java:178)
        at org.verapdf.parser.PDFParser.getXRefInfo(PDFParser.java:548)
        at org.verapdf.parser.PDFParser.getXRefInfo(PDFParser.java:221)
        at org.verapdf.io.Reader.init(Reader.java:149)
        at org.verapdf.io.Reader.<init>(Reader.java:60)
        at org.verapdf.cos.COSDocument.initReader(COSDocument.java:122)
        at org.verapdf.cos.COSDocument.<init>(COSDocument.java:89)
        at org.verapdf.pd.PDDocument.<init>(PDDocument.java:63)
        at org.verapdf.gf.model.GFModelParser.<init>(GFModelParser.java:83)
        at org.verapdf.gf.model.GFModelParser.createModelWithFlavour(GFModelParser.java:106)
        at org.verapdf.pdfa.VeraFoundry.createParser(VeraFoundry.java:80)
        at org.verapdf.pdfa.VeraFoundry.createParser(VeraFoundry.java:86)
        at org.verapdf.processor.ProcessorImpl.process(ProcessorImpl.java:115)
        at org.verapdf.processor.BatchFileProcessor.processItem(BatchFileProcessor.java:98)
        at org.verapdf.processor.BatchFileProcessor.processList(BatchFileProcessor.java:74)
        at org.verapdf.processor.AbstractBatchProcessor.process(AbstractBatchProcessor.java:102)
        at org.verapdf.cli.VeraPdfCliProcessor.processFilePaths(VeraPdfCliProcessor.java:172)
        at org.verapdf.cli.VeraPdfCliProcessor.processPaths(VeraPdfCliProcessor.java:131)
        at org.verapdf.cli.VeraPdfCli.singleThreadProcess(VeraPdfCli.java:138)
        at org.verapdf.cli.VeraPdfCli.main(VeraPdfCli.java:106)
        at org.verapdf.apps.GreenfieldCliWrapper.main(GreenfieldCliWrapper.java:54)
```

**Comment by palm72** [comment]
It is the same problem when the xref table contain offset values out of file range: "Destination offset is greater than stream length".  VeraPDF do not report which file that cause the crash.

**Comment by bdoubrov** [comment]
Fixed in the latest dev build 1.19.168

**Comment by bdoubrov** [comment]
Fix released in v1.20
